client/src/components/DogSelection.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Dog Selection Component
+ *
+ * Component for the "select a dog page"
+ * TODO: populate real dog data, handle selected meet and greet
+ *
+ * @summary    Component for the fosters page
+ * @author     Parth Patel
+ *
+ */
+
+/*
+Data is mapped to page from allDogs dictionary which is populated by 
+a dictionary of the following format. 
+{ 1: dog1, 2: dog2 ... 5: dog5}
+There can be at most 5 dogs in this dictionary.
+
+Each dog object (dog1, dog2...) has the following format:
+
+dog = {
+  name: [string],
+  image: [image variable from import],
+  curInterested: {
+    preference: [string],
+    time: [string],
+  },
+  notes: [string],
+  othersInterested: [
+    {
+      name: [string],
+      preference: [string],
+      time: [string],
+    },
+    {
+      name: [string],
+      preference: [string],
+      time: [string],
+    },
+  ],
+  active: [boolean],
+}
+
+*/
+
+import React, { useState } from "react";
+import styled from "styled-components";
+import DogSummary from "./DogSummary";
+import DogHappy from "../images/DogHappy.png";
+
+const dog1 = {
+  name: "Happy",
+  image: DogHappy,
+  curInterested: {
+    preference: "Love",
+    time: "11:00 PM",
+  },
+  notes:
+    "Lorem Ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ",
+  othersInterested: [
+    {
+      name: "Stacy",
+      preference: "Like",
+      time: "12:05 PM",
+    },
+    {
+      name: "Molly",
+      preference: "Love",
+      time: "1:42 PM",
+    },
+  ],
+  active: false,
+};
+const dog2 = dog1;
+const dog3 = dog1;
+const dog4 = dog1;
+const dog5 = dog1;
+// dummy data
+const allDogs = { 1: dog1, 2: dog2, 3: dog3, 4: dog4, 5: dog5 };
+
+const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const DogWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  overflow: scroll;
+  height: 660px;
+  @media screen and (max-width: 750px) {
+    width: 80%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  @media screen and (max-width: 350px) {
+    width: 100%;
+  }
+`;
+
+const ActiveWrapper = styled.div`
+  height: 630px;
+  width: 259px;
+  margin-left: 1vw;
+  margin-right: 1vw;
+  margin-bottom: 2vh;
+  padding: 10px 0px;
+  border-radius: 10px;
+  border: 3px solid #8dc442;
+  background: ${(props) => (props.active ? "rgba(141, 196, 66, 0.5)" : "white")};
+`;
+
+const Button = styled.button`
+  position: relative;
+  width: 216px;
+  height: 52px;
+  bottom: 0;
+  margin-top: 15px;
+
```

Comment:
```suggestion
  margin-top: 15px;
  border: none;
```
Confirm button has no border in designs